src/Main.java

```diff
@@ -143,7 +141,30 @@ public static void main(String[] args) {
                         patient.viewProfile();
                         break;
                     case 2:
-                        System.out.println("Update Profile");
+                        System.out.println("Fill in the fields you want to update only. Leave others empty");
+
+                        System.out.print("Enter firstname: ");
+                        String firstname = scanner.nextLine();
+
+                        System.out.print("Enter lastname: ");
+                        String lastname = scanner.nextLine();
+
+                        System.out.print("Enter Date of Birth: ");
+                        String DoB = scanner.nextLine();
+
+                        System.out.print("Enter HIV Status: ");
+                        String HIVStatus = scanner.nextLine();
```

Comment:
You should have added some data validation as in the CompleteRegistration method.